.github/workflows/test.yml

```diff
@@ -10,6 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 10
 
+    env:
+      SECRET_KEY: "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
```

Comment:
It's a security issue to hardcode sensitive information like `SECRET_KEY` in your configuration files. Consider using GitHub Secrets to store this value securely and reference it in your workflow.